scripts/install-native.mjs

```diff
@@ -56,7 +56,10 @@ import fs from 'fs-extra'
       path.join(tmpdir, 'package.json'),
       JSON.stringify(pkgJson)
     )
-    let { stdout } = await execa('yarn', ['--force'], { cwd: tmpdir })
+    await fs.writeFile(path.join(tmpdir, '.npmrc'), 'node-linker=hoisted')
```

Comment:
Should we add this to .npmrc permanently instead of writing here each time?

Reply:
I think we only want this behavior for the dummy package that gets created as part of this postinstall, which is ephemeral and deleted after the run completes